docs/tutorials/quick-start/index.rst

```diff
@@ -350,32 +350,87 @@ install cert-manager. This example installed cert-manager into the
 
 .. code-block:: shell
 
-    $ helm install --name cert-manager --namespace cert-manager stable/cert-manager
+    # Install the cert-manager CRDs. We must do this before installing the Helm
+    # chart in the next step
+    $ kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/v0.6.0-alpha.0/deploy/manifests/00-crds.yaml
```

Comment:
No qualms with switching the installation mode away from Helm, but this location does come with the burden of being very specific to a release, and doesn't look like it's well set to float through updates in the future. 

I don't know how solid the CRD mechanism is, but if you can include the same YAML manifests elsewhere where they get "updated with latest", it'll give more longevity to this PR update, which will otherwise need tweaking with every CRD release update.

Reply:
I'm assuming that there's more quirks with Helm not cleanly installing the CRDs - or related cleanup - pieces, which is why this is being installed outside/prior to the rest of the install using Helm?

Reply:
:+1: for now, I think we can change this to `release-0.6` so it is pinned to a minor version. Once we sort out some of our deployment artifact publishing, I think we can look at a 'latest' endpoint too 😄